### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://hosted.weblate.org/projects/trustroots/translation/) for Trustroots/translation.

### CODE REVIEW

**Comment by nicksellen** [comment]
> Would you merge as is, or should we squash the translated components for each person, or for each language, or everything to a single commit, or ... ?
> Also, it may be possible to set up weblate to have just one weblate component for the whole trustroots project, if there can be multiple translation files per component. That would simplify the history as well.
> Or we could have a separate repository for translations and not worry about its history; and let weblate organize its history as it wishes.

On these 3 questions, my opinion would be:

1. to squash merge the weblate pull requests on the basis the history is not very meaningful to store (as weblate has the job to manage the translations, I see the weblate as the source of truth, and this branch just the output that would put into the repo out of convenience for us) - do you see a use case for using the history if it is more seperated?
2. I did wonder about the use of "weblate components" (also wondering about the case if we had seperate "things" to translate (e.g. mobile app), but I don't see any compelling reason to change what we have right now
3. I would keep it in the same repo as adding more repos adds more complexity to manage them (they can get out of sync so would need some concept of how they relate to each other)

(the caveat to my opinions is they are all on the basis I personally don't care so much about git history in general, as long as there is some....)

**Comment by simison** [comment]
...pending of course that we don't have a possibility to inject malicious content/scripts via Weblate or anything like that. ;-)

File: public/locales/de/users.json [hunk 1]
```diff
@@ -1,43 +1,59 @@
...
+    "Your profile description should be longer so that you can send messages": {
+        "": ""
+    },
+    "female": "",
+    "male": "",
+    "non-binary": "",
+    "other": "",
+    "{{birthdate, age}} years": "",
+    "“Everyone is necessarily the hero of their own life story": {
+        "”": ""
```
Comment (mrkvon) [inline public/locales/de/users.json:57]: Apparently, we have some issues with settings here.

File: public/locales/cs/offers.json [hunk 2]
```diff
@@ -1,22 +1,24 @@
 {
   "Accommodation": "Můj dům",
-  "At most {{count}} guests._0": "Nejvíce jeden host.",
-  "At most {{count}} guests._1": "Nejvíce {{count}} hosté.",
-  "At most {{count}} guests._2": "Nejvíce {{count}} hostů.",
+  "At most {{count}} guests": {
+    "At most {{count}} guests._0": "Nejvíce jeden host.",
+    "At most {{count}} guests._1": "Nejvíce {{count}} hosté.",
+    "At most {{count}} guests._2": "Nejvíce {{count}} hostů."
+  },
```
Comment (nicksellen) [inline public/locales/cs/offers.json:7]: Ah, I didn't handle this quite right.

File: public/locales/es/messages.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
-  "...explaining that you are interested in meeting them, not just looking for free accommodation.": "",
```
Comment (nicksellen) [inline public/locales/es/messages.json:2]: Needs fixing too...

**Comment by mrkvon** [comment]
@nicksellen How do you commit to the weblate fork? Do we have rights to do that?

**Comment by nicksellen** [comment]
> @nicksellen How do you commit to the weblate fork? Do we have rights to do that?

@mrkvon ah yeah, interesting, I didn't think about it much actually, I added the weblate repo as another remote (`git remote add weblate git//whatever/their/repo/is`), checked out the branch, and pushed. I guess it went to their repo :/

And I think if I fix those issues I pointed out then get in there quick with the merge (before weblate notices and force pushes again!)

**Comment by nicksellen** [comment]
@mrkvon or maybe I didn't push to that branch at all, I'm a bit pre-occupied at the moment and did the final bit very quickly!

**Pull request merged**